neurom/stats.py

```diff
@@ -176,3 +176,24 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
+    '''
+    Parameters:
+        paired_dats: a list of tuples or where each tuple
+                         contains the paired data lists from two datasets
+
+    Options:
+        p : integer that defines the order of p-norm
+        test: Stat_tests\
+            Defines the statistical test to be used, based\
+            on the scipy available modules.\
+            Accepted tests: ks_2samp, wilcoxon
+
+    Returns:
+        A float corresponding to the p-norm of the scores that have
```

Comment:
You should also make sure you explain that "0" is high similarity and "1" is low similarity.